doc/ref_model/chapters/chapter08.rst

```diff
@@ -202,13 +202,30 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
+As a consumer of cloud services to support internal Business operations and BSS/OSS, the focus is on meeting the needs of the organisation's applications . Historically this came with the need to operate and support the organisation's infrastructure needs. The result was a split of the CIO organisation into Delivery and Operations groups. At the same time that the CIO application workloads are moving to SaaS and other Cloud Providers, the CTO Network Systems are migrating from running on custom dedicated infrastructure to run on virtualised COTS infrastructure; examples include IMS, 3GPP (4G & 5G) functions, IP Routers and Firewalls are being provided as VNFs and CNFs.  These network workloads are now also being deployed on private CSP clouds as well as public clouds.
+
+As outlined in section "8.2.2 Telco Cloud", the result is that the CSP "network" is now an interconnected set of distributed Cloud Infrastructure supported by different Cloud Providers, including the CSP, and, hence, the term "Hybrid Multi-Cloud", and the need for the CSP to be able to support and utilize this interconnected cloud is both inevitable and essential.
+
+As a consumer and provider of Cloud Services, the CSP will continue to need to build and manage its own Cloud Infrastructure as well as provide: 
+
+- cloud orchestration solutions to orchestrate the utilisation of cloud services and capabilities from its own and other Cloud Providers;
+- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks. 
+
+The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions. The identification of existing appliable standards and gaps across the interactions needs to be completed. As a first step, the following criteria for inclusion of a standard/technology is defined. These standards/technologies must:
+
+- provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions
+- be already mature Open Standards that have either been adopted or nurtured by recognised bodies with the telecommunications industry (e.g. ITU, ETSI, TMForum, GSMA, 3GPP, ISO and national Standards Organiations , (ANSI etc,) NIST)
+- have reference implementations or an active open source project/s or consortia providing implementations (e.g., CNCF. Open Infra)
+- allow the CSP to source delivery and support services based on these from multiple vendors
+- allow the CSP to actively contribute to and request capabilities / coverage of the standard / technology
+- not  be the sole proprietary property of a vendor / company
+- not be focused on "Transactions / Conversations" or "User / Data Plane" standards (typically IETF, IEEE, MEF / Carrier Ethernet etc.)
```

Comment:
Not sure how to deal with requirements against standards. RM should set requirements against platforms. So a requirement in RM could be to follow certain standards which provide capabilites...., ..., and give a list of the standards to follow. But requiring something from an SDO, I don't know.

Reply:
@ulikleber initial ambition was to populate table of applicable "standards", that would be recommended towards helping CSP to meet objectives of standing up and operating a multi-cloud environment and identify gaps that need to be addressed.  For this release (moselle) the ambition has been curtailed to just getting the criteria for inclusion of such "standards" (which was identified as required toward the end of prior release (lakelse)). If we can get to agreement on what is already covered (beyond OpenStack & Kubernetes) and what is missing, then we can agree what should be approach to handling the "gap" items. I agree that Anuket cannot mandate that some organisation should take on responsibility of covering the gap. I think in identify the gap that Anuket (RM) could help elaborating what this is and provide statement of requirement/need which can be use as input to CSP to "consider" (and maybe even tackle ;-) ). So discussion point rather than resolution.